features/user_can_see_a_sidebar.feature

```diff
@@ -0,0 +1,10 @@
+Feature: User can see a sidebar
+  As a User
+  In order to be able to read and navigate the site
+  I would like to have content and navigating tools on the index
+
+  Scenario: User can see a sidebar
+    Given I am on the "Index" page
+    Then I should see "Sports"
```

Comment:
You can DRY this scenario by making use of a table as follow

```gherkin
Scenario: User can see a sidebar and its content
  Given I am on the "Index" page
  Then I should see:
    | content |
    | Sports  |
    | Fashion |
    | Tech    |
```

Then in your step definition, you'd have something like

```ruby
# [...] step title here do |table|
  table.hashes.each { |_k, content| expect(page).to have_content content }
# end
```